src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public enum Color {
+    WHITE("White"),
+    GREEN("Green"),
+    PINK("Pink"),
+    BLACK("Black"),
+    BLUE("Blue"),
+    RED("Red");
+
+    private final String value;
+
+    Color(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
```

Comment:
Good point. But you either can use `Color.name().toLowerCase()`, see next comment